tests/unit/api/c/dmx_merger/test_dmx_merger.cpp

```diff
@@ -432,6 +434,7 @@ TEST_F(TestDmxMerger, AddSourceErrNoMemWorks)
   TestAddSourceMemLimit(true);
 }
 
+#if 0  // TODO: Update this so it uses the new update functions
```

Comment:
Do you plan on updating these tests in this PR or another one?

Reply:
Another one, this one will be rebased and merged momentarily.